### ISSUE TITLE
How to calculate the test Agreement?

### ISSUE THREAD

**Issue by Alan-JW** [samuelstanton/gnosis#16]
Hi，@PolinaKirichenko ，How is the test Agreement in your paper calculated in the code? thanks for your answer！

**Comment by samuelstanton** [comment]
we calculate agreement as
```
agreement = teacher_logits.argmax(-1).eq(student_logits.argmax(-1)).float().mean()
```

**Comment by samuelstanton** [comment]
if you're using a teacher ensemble you'll want to softmax the logits and average across the ensemble dimension first
